ngraph/core/reference/src/runtime/reference/fft.cpp

```diff
@@ -482,6 +482,21 @@ namespace ngraph
 
                     return result;
                 }
+
+                std::vector<int64_t> lengths_except_given_axis(const std::vector<int64_t>& v,
+                                                               int64_t axis)
+                {
+                    std::vector<int64_t> result;
+                    int64_t num_of_elems = static_cast<int64_t>(v.size());
+                    for (int64_t i = 0; i < num_of_elems; ++i)
+                    {
+                        if (i != axis)
+                        {
+                            result.push_back(v[i]);
+                        }
+                    }
+                    return result;
+                }
```

Comment:
I think we can remove this function and just use `std::vector::erase()` method.
For example:
```
std::vector<int64_t> v;
v.erase(v.begin() + axis);
```

Reply:
We need to return the **copy** of  `v` (see usage this function), except the element with the index `axis`. Hence, we cannot remove this function. But the body of the function `lengths_except_given_axis` can be rewritten using `std::remove`.

Reply:
Reverted changes in the function `lengths_except_given_axis`: we have segfault with such changes.

Reply:
I disagree with this method. I hope we can investigate why new better implementation introduced a segfault and fix it.

Reply:
The original comment is still here. We shouldn't reinvent the wheel here if stl already has the method which we need.

Reply:
Fixed.